### ISSUE TITLE
'~ files really needed?

### ISSUE THREAD

**Issue by dulhaver** [horald/phpmysync#1]
or can they they be ignored in the .gitignore, with an entry like ...
 
`*~`

**Comment by horald** [comment]
yes, they can ignored in the .gitignore. Sorry my .gitignore is not updated.
